src/lib/db/schema.ts

```diff
@@ -78,6 +87,9 @@ export const verificationTokens = pgTable(
 
 export const gameSessions = pgTable("gameSession", {
   id: serial("id").primaryKey(),
+  gameSessionScheduleId: integer("gameSessionScheduleId")
+    .notNull()
+    .references(() => gameSessionSchedules.id, { onDelete: "cascade" }),
```

Comment:
I think for our use case, we actually want the gameSession record to remain even if the schedule is deleted. Can you look into we we can not cascade on delete without violating foreign key constraints?